lib/mocha.js

```diff
@@ -143,7 +144,9 @@ Mocha.prototype.reporter = function(reporter, reporterOptions){
   } else {
     reporter = reporter || 'spec';
     var _reporter;
-    try { _reporter = require('./reporters/' + reporter); } catch (err) {}
+    // Try to load a built-in reporter.
+    if (reporters[reporter]) _reporter = reporters[reporter];
```

Comment:
dunno.  I wanted to eliminate them from the core except for `spec`, so whatever's clever for now.